docs/plans/2026-05-14-m24-lifecycle-contract-layer.md

```
@@ -0,0 +1,335 @@
+* Build the same fixture vault as `test_ops_lifecycle`.
+* Run `ovp-ops-state --rebuild`.
+* Assert row count, state distribution, idempotency.
+* Assert that the projection answer matches the kernel answer for
+  every item (the projection cannot lie about state relative to the
+  kernel).
+
+### Stage 1.C — CLI surface
+
+Two new entry points in `pyproject.toml`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)